web-console/3-project-details/jobs.md

```diff
@@ -0,0 +1,59 @@
+
+
+# Jobs
+
+- A Job creates one or more pods which run to completion, i.e. they don't run continuously like deployments.
+- A Cron Job extends a regular Job by allowing users to schedule when a job will run.
+- This documentation uses "Job" to refer to both Jobs and Cron Jobs, except when Cron Jobs are called out specifically.
+
+## Overview
+- Jobs and Cron Jobs would be listed under a "Jobs" heading in the Application and Resource views of the Overview.
+- The heading over individual Job names would specify if it is a Job or Cron Job.
+- Jobs (and Cron Jobs) that have an app label should be grouped with that application in the Application view.
```

Comment:
I'm not sure if I understand how something will be grouped as an app and a job. Do you have an image/ mock for this? I might not be understanding the statement correctly.

Reply:
I'll clarify that. I think actually that Jobs should be in Other Resources if they don't have an app label. Will fix.